google/cloud/bigtable/internal/table_test.cc

```diff
@@ -67,6 +67,19 @@ struct MockRpcFactory {
 
 }  // anonymous namespace
 
+TEST_F(NoexTableTest, ChangeOnePolicy) {
+  bigtable::noex::Table table(client_, "some-table",
+                              bigtable::AlwaysRetryMutationPolicy());
+  EXPECT_THAT(table.table_name(), ::testing::HasSubstr("some-table"));
+}
+
+TEST_F(NoexTableTest, ChangePolicies) {
+  bigtable::noex::Table table(client_, "some-table",
+                              bigtable::AlwaysRetryMutationPolicy(),
+                              bigtable::LimitedErrorCountRetryPolicy(42));
+  EXPECT_THAT(table.table_name(), ::testing::HasSubstr("some-table"));
```

Comment:
Similarly here: this is not testing either the always-retry-until-N-errors policy. Should that be done here, or is that out of scope?

Reply:
See above.